Makefile.eve

```diff
@@ -25,15 +20,16 @@ help: Makefile
 	@echo "  docker-tag-gcc: print docker tag for gcc kernel"
 	@echo "  docker-tag-clang: print docker tag for clang kernel"
 	@echo
-	
+
 kernel-%: Dockerfile.%
 	@echo "Building kernel version $(BRANCH):$(VERSION)-$* with compiler $*"
 	docker buildx build \
 	--build-arg="SOURCE_DATE_EPOCH=$(SOURCE_DATE_EPOCH)" \
 	--build-arg="KBUILD_BUILD_TIMESTAMP=$(KBUILD_BUILD_TIMESTAMP)" \
-	--platform $(PLATFORM) -t $(BRANCH):$(VERSION)-$* --load -f Dockerfile.$* .
+	--platform $(PLATFORM) -t lf-edge/eve-kernel:$(BRANCH)-$(VERSION)-gcc --load -f Dockerfile.$* .
 
 docker-tag-%:
-	@echo "docker.io/lfedge/eve-kernel:$(BRANCH)-$(VERSION)$(DIRTY)-$*"
-
+	@echo "docker.io/lfedge/eve-kernel:$(BRANCH)-$(VERSION)$(DIRTY)-gcc"
```

Comment:
We seem to have lost the setting of DIRTY.
Furthermore, we should probably keep $DIRTY in the tag all the time to make sure we can not accidentally push something built from dirty to docker hub.

I don't know if it makes sense to do this by setting VERSION to git describe --match '$$^' --abbrev=8 --always --dirty

Reply:
@yash-zededa please rebase to my latest Makefile.eve. And why did you replace $* with gcc ?

Reply:
@yash-zededa and please squash all commits as @eriknordmark mentioned

Reply:
@mikem-zed Yes, I will squash all the commits. Added the `$*`

Reply:
@mikem-zed @eriknordmark commits are squashed and signed-off. The history is now clean :)